lib/test/t_ComposedDistribution_std.cxx

```diff
@@ -246,6 +247,35 @@ int main(int, char *[])
     anotherSample = distribution.getSample(size);
     fullprint << "anotherSample mean=" << anotherSample.computeMean() << std::endl;
     fullprint << "anotherSample covariance=" << anotherSample.computeCovariance() << std::endl;
+    // Create and print a complex distribution
+    ComposedDistribution::DistributionCollection aCollection2;
+    Normal marginalN(0.0, 1.0);
+    marginalN.setName("First");
+    Description componentN(1);
+    componentN[0] = "One";
+    marginalN.setDescription(componentN);
+    aCollection2.add( Distribution(marginalN) );
+    aCollection2[0].setName("First");
```

Comment:
Each component of the collection has a specific `name`, which is an attribute of the `Collection`. This might be different from the description of each marginal, which is a property of a `Distribution`. This name should be printed before the name of each marginal.

Reply:
This is wrong. Name is a property of a `Persistent` object. It is used by the `Study` to load an object that has been saved on disk. There is no need to print that information at that level.